echopop/workflow/feat_hake.py

```diff
@@ -996,6 +1002,14 @@
     df_kriged_abundance_table.to_pickle(FILES_DIR / "df_kriged_abundance_table.pkl")
     # Biomass table - kriged data
     df_kriged_biomass_table.to_pickle(FILES_DIR / "df_kriged_biomass_table.pkl")
+    # Stratified results - transect data
+    transect_results.to_pickle(FILES_DIR / "stratified_transect_results.pkl")
+    # Stratified results - kriged data
+    kriged_results.to_pickle(FILES_DIR / "stratified_kriged_results.pkl")
+    # Stratified replicates - transect data
+    transect_replicates.to_pickle(FILES_DIR / "stratified_transect_replicates.pkl")
+    # Stratified replicates - kriged data
+    kriged_replicates.to_pickle(FILES_DIR / "stratified_kriged_replicates.pkl")
```

Comment:
Suggest adding `jh_` as prefix to the `*_results` and `*_replicates` variables here for clarity.